airflow/providers/apache/hive/operators/hive_stats.py

```diff
@@ -67,17 +68,26 @@ def __init__(self,
                  table: str,
                  partition: str,
                  extra_exprs: Optional[Dict] = None,
-                 col_blacklist: Optional[List] = None,
+                 excluded_columns: Optional[List] = None,
                  assignment_func: Optional[Callable[[str, str], Optional[Dict]]] = None,
                  metastore_conn_id: str = 'metastore_default',
                  presto_conn_id: str = 'presto_default',
                  mysql_conn_id: str = 'airflow_db',
                  *args, **kwargs) -> None:
+        if 'col_blacklist' in kwargs:
```

Comment:
This will still fail precommit check I think, we can exlcude this file or add some kind of "#ignore" that we can check in pyprep in precommit. Or ezclude this file in that precommit check

Reply:
It is excluded in precmmit :)

Reply:
Oh yes, thank You. 🙏